s2p/__init__.py

```diff
@@ -644,7 +656,16 @@ def main(user_cfg, start_from=0):
         if cfg['max_processes_stereo_matching'] is not None:
             nb_workers_stereo = cfg['max_processes_stereo_matching']
         else:
-            nb_workers_stereo = nb_workers
```

Comment:
why are you changing this in this PR?

Reply:
Yup, I accidentally used the wrong s2p version